docs/javascript/example/butt-cast.md

```diff
@@ -0,0 +1,46 @@
+<code-explorer>
```

Comment:
this is the custom tag defined in docsify config in index.html which transforms the contents into syntax highlighted code , with comments which can be seen by hovering/  clicking on lines